packages/did-provider-ion/package.json

```diff
@@ -19,6 +19,7 @@
     "@veramo/key-manager": "^4.1.1",
     "@veramo/kms-local": "^4.1.1",
     "base64url": "^3.0.1",
+    "canonicalize": "^1.0.8",
```

Comment:
I think this dependency should be declared in the `@veramo/credential-w3c` package

Reply:
@mirceanis I just had noticed that did-provider-ion used this dependency but didn't declare it explicitly. I also added it in credential-w3c package

Reply:
You're right. I hadn't noticed 👓